kong/plugins/statsd/statsd_logger.lua

```diff
@@ -17,6 +17,8 @@ function statsd_mt:new(conf)
   end
   
   local statsd = {
+    -- FIXME: Schema migration may be better, but converting original json is not easy.
+    prefix = conf.prefix or "kong.",
```

Comment:
Updating plugin configs via migration is not terribly hard when you use the DAO abstraction (for example, https://github.com/Mashape/kong/blob/master/kong/plugins/cors/migrations/postgres.lua).